### ISSUE TITLE
Clearer unread feed indicator in dark mode

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#3659]
https://forums.zotero.org/discussion/111415/feature-request-zotero-7-beta-alternative-formatting-for-unread-rss-feeds

Not sure if a heavier bold is reliably available?

**Comment by yexingsha** [comment]
Yes, this is because `font-weight: 600` on Segoe UI is just not thick enough. As a quick fix, we should change every `font-weight: 600` to `font-weight: bold`, but ideally we want to only change this for windows.

**Comment by marbris** [comment]
I'd like to bump this issue, but for Linux. I'm using Zotero 7 on Ubuntu. The read/unread in the feeds are difficult to tell apart with the weak bold. I use the JetBrainsMono font, so it's not a standard font. But it'd be useful if read/unread was more clearly distinguished, either by stronger bold or by color.
